src/zc/buildout/testing.py

```diff
@@ -541,3 +564,37 @@ def _normalize_path(match):
     re.compile(
     'Not upgrading because not running a local buildout command.\n'
     ), '')
+
+def run_buildout(command):
+    # Make sure we don't get .buildout
+    os.environ['HOME'] = os.path.join(os.getcwd(), 'home')
+    args = command.strip().split()
```

Comment:
Can be simplified to `command.split()`, since `.split()` with no arguments implicitly strips.